### PR TITLE
feat/tailwind buttons

### PR DESCRIPTION
**Pull request by tenthwalker**
What I did:
Did this break anything?
- [x] No
- [ ]  Yes
Type of change
- [ ] New feature (non-breaking change which adds functionality)
- [x]  Styling
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.
Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing
What's next?

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -9,4 +9,6 @@ module.exports = withMT({
     extend: {},
   },
   plugins: [],
-});
\ No newline at end of file
+});
+
+className="h-12 min-w-[8rem] rounded-lg border-2 border-emerald-600 bg-emerald-500 text-emerald-50 shadow-lg hover:bg-emerald-600 focus:outline-none focus:ring focus:ring-emerald-600"
```
Comment (tenthwalker) [inline tailwind.config.js:14]: Does this styling in the config file get called elsewhere?

File: src/ButtonRule.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const CustomButton = ({ children, ...props }) => {
```
Comment (tenthwalker) [inline src/ButtonRule.js:3]: You cooked on this, well done

**Review by tenthwalker — COMMENTED** [review]
Thanks for getting these updated, I'm a fan of the high contrast and the hover state!

**Review by JoeyStewart — APPROVED** [review]

File: src/ButtonRule.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <button
+      type="button"
+      className="text-white bg-gradient-to-r from-black to-gray-700 hover:bg-gradient-to-bl focus:ring-4 focus:outline-none focus:ring-gray-500 dark:focus:ring-gray-800 font-medium rounded-lg text-sm px-5 py-2.5 text-center me-2 mb-2"
+      {...props}
+    >
+      {children}
+    </button>
```
Comment (JoeyStewart) [inline src/ButtonRule.js:11]: This is really cool way of standardization.

**Review by JoeyStewart — APPROVED** [review]

**Pull request merged**